jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/FileLock.java

```diff
@@ -82,24 +84,26 @@ public boolean lockFile(boolean blocking, String callSite) throws IOException {
 		if (!locked && blocking) { /* try again, this time with a blocking lock and a timeout */
 			FileLockWatchdogTask wdog = new FileLockWatchdogTask();
```

Comment:
Can this be initialized only inside the `if (useWatchdog)` block?

Reply:
No, there is another `if (useWatchdog)` block, and reference of `wdog` at L126.

Reply:
I saying to leave the declaration where it is, but only do `wdog = new FileLockWatchdogTask();` inside the if.

Reply:
This is the last thing I'm waiting for...

Reply:
Ah, somehow missed that notification, changed it.